xbmc/cores/paplayer/VideoPlayerCodec.cpp

```diff
@@ -29,7 +29,10 @@ VideoPlayerCodec::VideoPlayerCodec()
   m_nAudioStream = -1;
   m_nDecodedLen = 0;
   m_bInited = false;
+  m_bCanSeek = false;
   m_pResampler = NULL;
+  m_audioFrame = {};
+  m_planes = 0;
```

Comment:
Another style nit, we prefer to initialize member variables in the header file with the same brace initialization, because it works for all constructors.